From bbe2c737eb2b62035424552822bc13be532927e3 Mon Sep 17 00:00:00 2001
From: Aaradhana Sahu <quic_aarasahu@quicinc.com>
Date: Wed, 10 Aug 2022 13:15:45 +0530
Subject: [PATCH] ath12k: move info print to dbg print

This patch moves unwanted "ath12k: tx queue params cached since
vif is not assigned" warning to debug as this may affect debugging

So, moved ath12k info print to ath12k dbg print

Signed-off-by: Aaradhana Sahu <quic_aarasahu@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/mac.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath12k/mac.c b/drivers/net/wireless/ath/ath12k/mac.c
index f119e8f..04552a5 100644
--- a/drivers/net/wireless/ath/ath12k/mac.c
+++ b/drivers/net/wireless/ath/ath12k/mac.c
@@ -5671,7 +5671,8 @@ static int ath12k_mac_op_conf_tx(struct ieee80211_hw *hw,
 	 */
 	mutex_lock(&ah->conf_mutex);
 	if (!arvif->is_created) {
-		ath12k_info(NULL, "tx queue params cached since vif is not assigned to radio\n");
+		ath12k_dbg(NULL, ATH12K_DBG_MAC,
+			"tx queue params cached since vif is not assigned to radio\n");
 		arvif->cache.tx_conf.changed = true;
 		arvif->cache.tx_conf.ac = ac;
 		arvif->cache.tx_conf.tx_queue_params = *params;
-- 
2.17.1

